ebs/ebs.go

```diff
@@ -423,7 +434,7 @@ func (d *Driver) DeleteVolume(req Request) error {
 		log.Debugf("Detached %v(%v) from %v", id, volume.EBSID, volume.Device)
 	}
 
-	if !referenceOnly {
+	if !referenceOnly && !volume.PreExist {
```

Comment:
I thought if the volume created by us, we should delete it, right ? This is for distinguishing that case.